collections/src/main/scala/strawman/collection/package.scala

```diff
@@ -104,7 +104,7 @@ class LowPriority {
   import strawman.collection._
 
   /** Convert array to iterable via view. Lower priority than ArrayOps */
-  implicit def arrayToView[T](xs: Array[T]): ArrayView[T] = ArrayView[T](xs)
+  implicit def arrayToImmutableArray[T](xs: Array[T]): immutable.ImmutableArray[T] = immutable.ImmutableArray.wrapArray[T](xs)
```

Comment:
Here too, we should use `collection.IndexedSeq[T]` as a return type instead of `immutable.ImmutableArray[T]`